bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -405,6 +405,10 @@ export class HasProps extends Backbone.Model
       # this skips optional properties like radius for circles
       if (prop.optional || false) and prop.spec.value == null and (name not of @_set_after_defaults)
         continue
+      if prop.spec.transform
+        if not prop.spec.transform._listeners[this._listenId]
```

Comment:
It would be better not to dig into private Backbone API. What is this trying to accomplish? Is this to prevent the same transform from being listened to multiple times?